packages/esm-patient-registration-app/src/patient-registration/field/address/address-hierarchy.component.tsx

```diff
@@ -78,17 +85,27 @@ export const AddressHierarchy: React.FC = () => {
           width: '50%',
           paddingBottom: '5%',
         }}>
-        {addressLayout.map((attributes, index) => (
-          <ComboInput
-            key={`combo_input_${index}`}
-            name={attributes.name}
-            labelText={attributes.labelText}
-            id={attributes.name}
-            placeholder={attributes.labelText}
-            setSelectedValue={setSelectedValue}
-            selected={selected}
-          />
-        ))}
+        {useAddressHeirarchy
+          ? addressLayout.map((attributes, index) => (
+              <ComboInput
+                key={`combo_input_${index}`}
+                name={attributes.name}
+                labelText={t(attributes.name)}
```

Comment:
This works, but is missing the fallback text for when `eact-i18next` fails. Do you think its reasonable to add this here and for the static version as well?

Reply:
We have defined the translation keys a few lines above this one, hence the keys and values for the address fields are stored in the translation file and is taken for the specific key.

Reply:
Right, but we've seen that translation file be ignored if react-i18next isn't loaded properly. Our site-wide syntax for this as far as i am aware is `t("variableName", "Fallback Text")`. What do you think?